mscore/bendcanvas.cpp

```diff
@@ -40,62 +42,75 @@ BendCanvas::BendCanvas(QWidget* parent)
 
 void BendCanvas::paintEvent(QPaintEvent* ev)
       {
-      int w = width();
-      int h = height();
-
-      QPainter p(this);
-      p.fillRect(rect(), Qt::white);
-
-      static const int ROWS    = 13;
-      static const int COLUMNS = 13;
-
-      int xs = w / COLUMNS;
-      int ys = h / ROWS;
-      int lm = xs / 2;
-      int tm = ys / 2;
-      int tw = (COLUMNS - 1) * xs;
-      int th = (ROWS - 1)    * ys;
-
-      QPen pen = p.pen();
+      // not qreal here, even though elsewhere yes,
+      // because width and height return a number of pixels,
+      // hence integers.
+      const int w = width();
+      const int h = height();
+
+      const qreal columnWidth = qreal(w) / COLUMNS;
+      const qreal rowHeight = qreal(h) / ROWS;
+
+      // let half a column of margin around
+      const qreal leftPos = columnWidth * .5; // also left margin
+      const qreal topPos = rowHeight * .5;    // also top margin
+      const qreal rightPos = w - leftPos; // right end position of graph
+      const qreal bottomPos = h - topPos; // bottom end position of graph
+
+      QPainter painter(this);
+      painter.setRenderHint(QPainter::Antialiasing);
+      // #2c2c2c is musescore's "dark color", so a little brighter
+      painter.fillRect(rect(), preferences.isThemeDark() ? QColor("#3c3c3c") : Qt::white);
```

Comment:
`#2c2c2c` or `#3c3c3c`? If it's the general dark colour, shouldn't there be a variable to represent it?

Reply:
It is in `palette_dark_fusion.json`
I wonder if that is accessible. Let me check.